recipes/pyorgmode/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1" %}
+
+package:
+  name: pyorgmode
+  version: {{ version }}
+
+source:
+  fn: PyOrgMode-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/4f/81/e403d3c062000481e5d0b54167369a7b4d33a6e001ccc441bbefeb611600/PyOrgMode-0.1.tar.gz
```

Comment:
This is very fragile for updating. Please change this to the following (this requires templating `name`).

```yaml
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

xref: https://github.com/conda-forge/staged-recipes/blob/a0222417ed30935a383f5773ff21d23322247cdd/recipes/example/meta.yaml#L18